circuit_knitting/utils/transforms.py

```diff
@@ -40,7 +45,14 @@
 
 
 class SeparatedCircuits(NamedTuple):
-    """Named tuple for result of :class:`separate_circuit`."""
+    """Named tuple for result of :func:`separate_circuit`.
+
+    ``subcircuits`` is a dict of circuits, keyed by each partition label.
+    ``qubit_map`` is a list of length number of qubits in the original circuit.
```

Comment:
```suggestion
    ``qubit_map`` is a list with length equal to the number of qubits in the original circuit.
```

Reply:
Oh man, thank you.  I rewrote this countless times and was never quite satisfied.